automation/taskcluster/lib/tasks.py

```diff
@@ -8,143 +8,110 @@
 
 
 class TaskBuilder(object):
-    def __init__(self, task_id, repo_url, branch, commit, owner, source, scheduler_id, tasks_priority='lowest'):
+    def __init__(self, task_id, repo_url, branch, commit, owner, source, scheduler_id, build_worker_type, beetmover_worker_type, tasks_priority='lowest'):
         self.task_id = task_id
         self.repo_url = repo_url
         self.branch = branch
         self.commit = commit
         self.owner = owner
         self.source = source
         self.scheduler_id = scheduler_id
+        self.build_worker_type = build_worker_type
+        self.beetmover_worker_type = beetmover_worker_type
         self.tasks_priority = tasks_priority
 
-    def raw_task(self, name, description, command, dependencies=[],
-                   artifacts={}, scopes=[], routes=[], features={},
-                   worker_type='github-worker'):
-        created = datetime.datetime.now()
-        expires = taskcluster.fromNow('1 year')
-        deadline = taskcluster.fromNow('1 day')
+    def craft_build_like_task(
```

Comment:
I'm having issues parsing this name, is it:
* Craft a build _like_ (e.g.: similar to) a task?
* Craft a task that's build-like (e.g.: very close to a build task)?

Reply:
Good call. That's the latter. I then change the name to `craft_build_ish_task()`